pkg/runtime/wfengine/component.go

```diff
@@ -22,10 +22,8 @@ import (
 
 	"github.com/microsoft/durabletask-go/api"
 	"github.com/microsoft/durabletask-go/backend"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"github.com/dapr/components-contrib/workflows"
-	componentsV1alpha1 "github.com/dapr/dapr/pkg/apis/components/v1alpha1" // This will be removed
+	"github.com/dapr/components-contrib/workflows" // This will be removed
```

Comment:
why will the import be removed?

Reply:
I don't know. Seems to be a weird copy/paste error inheriting the code comment from the removed line in the diff. Removed.